src/api/categories.ts

```diff
@@ -0,0 +1,18 @@
+import axios from "axios";
+import { Category } from "../provider/Transaction";
+import { getToken } from "../service/utils";
+import { apiUrl } from "./api";
+
+export type CategoriesResponseData = {
+  categories: Category[];
+};
+
+export const getCategories = () => {
+  return axios
+    .get<CategoriesResponseData>(`${apiUrl}/categories`, {
+      headers: {
```

Comment:
seems like this options object can be moved, it's the same and used everywhere now